api/type.rb

```diff
@@ -593,7 +593,7 @@ def get_type(type)
     def property_ns_prefix
       [
         'Google',
-        @__resource.__product.api_name.camelize(:upper),
```

Comment:
I believe this was a bug in all providers that used this type system that was never surfaced because they didn't support Cloud Build.

Basically, it was turning "Cloud Build" to Cloudbuild rather than CloudBuild for module names on the ruby classes.

Reply:
Doesn't look like it affects anyone else, so I guess we're fine.

We should have a MM-wide conversation about what to do with these 'property_class' methods (which is what this is used for). It feels weird to have them in api/.